cli/pkg/local/app.go

```diff
@@ -46,7 +46,9 @@ const (
 const (
 	DefaultInstanceID = "default"
 	DefaultOLAPDriver = "duckdb"
-	DefaultOLAPDSN    = "stage.db"
+	DefaultOLAPDSN    = "main.db"
+	DefaultMetaStore  = "meta.db"
```

Comment:
We should call this var `DefaultCatalogStore` since that's what it is (the metastore will still be in-memory – the name `meta.db` is mainly for user-understanding if they peek in the directory)